person.rb

```diff
@@ -19,4 +21,8 @@ def age_of?
   def can_use_service?
     age_of? || @parent_permission
   end
+
+  def validate_name
+    /[[:upper:]]/.match(@name)
+  end
```

Comment:
- [ ] The validate name method must use the corrector instance to validate and save the corrected name, please do so to comply with the project requirements.